src/jsontestrunner/main.cpp

```diff
@@ -111,8 +111,8 @@ static void printValueTree(FILE* fout, Json::Value& value,
     Json::Value::Members members(value.getMemberNames());
     std::sort(members.begin(), members.end());
     Json::String suffix = *(path.end() - 1) == '.' ? "" : ".";
-    for (auto name : members) {
-      printValueTree(fout, value[name], path + suffix + name);
+    for (const auto& name : members) {
+      printValueTree(fout, value[name], path + suffix.append(name));
```

Comment:
Wait, this didn't come from clang-tidy did it?
The suffix grows on every iteration of this loop now.
That isn't what the code was doing before.

Reply:
The const did.

The append did not. clang-tidy did not autofix it but suggested std::append().

Reply:
warning: string concatenation results in allocation of unnecessary temporary strings; consider using 'operator+=' or 'string::append()' instead [performance-inefficient-string-concatenation]